File: src/client/app/entities/Subject.js
export default class Subject {
  constructor(id, name) {
    this._id = id;
    this._name = name
  }

  get id() { return this._id };
  set id(value) { this._id = value }
  get name() { return this._name };
  set name(value) { this._name = value }

  toJSON() {
    return {
      id: this._id,
      name: this._name
    }
  }
}

File: TutoringTrainApp/app/src/main/java/at/train/tutorial/tutoringtrainapp/Data/Subject.java
package at.train.tutorial.tutoringtrainapp.Data;

import com.fasterxml.jackson.annotation.JsonView;

/**
 * Created by Moe on 12.01.2018.
 */

public class Subject {
    @JsonView({Views.Entry.In.loadNewest.class})
    private int id;
    @JsonView({Views.Entry.In.loadNewest.class})
    private String name;
    @JsonView({Views.Entry.In.loadNewest.class})
    private boolean isActive;
    @JsonView({Views.Entry.In.loadNewest.class})
    private String dename;
    @JsonView({Views.Entry.In.loadNewest.class})
    private String enname;

    @Override
    public String toString() {
        return "++++++++++++++++++++++++++++++++++++++++++++++++ " + id + " " + dename;
    }

    public int getId() {
        return id;
    }

    public String getName() {
        return name;
    }

    public boolean isActive() {
        return isActive;
    }

    public String getDename() {
        return dename;
    }

    public String getEnname() {
        return enname;
    }
}
